bor-mainnet: `./cmd/integration` bridge nil-ptr

command ` go run ./cmd/integration stage_exec  --datadir=/Users/alex.sharov/data/amoy --chain=amoy --batchSize=1m`


```
panic: interface conversion: bridge.PolygonBridge is nil, not *bridge.Bridge

goroutine 1 [running]:
github.com/ledgerwatch/erigon/polygon/bor.(*Bor).CommitStates(0xc000ebc1e0?, 0xc000001420?, 0xdc0094582c8?, {{0x1087b4e50?, 0xdc004542580?}, {0x1087b68e8?, 0xc000ebc1e0?}}, 0xdc008acd500?)
	github.com/ledgerwatch/erigon/polygon/bor/bor.go:1488 +0xb1b
github.com/ledgerwatch/erigon/polygon/bor.(*Bor).Finalize(0xc000ebc1e0, 0x1086d7a60?, 0xdc0094582c8, 0xc000001420, {0xdc008397210?, 0x106e95c13?, 0x1086d7c60?}, {0x0, 0x0, 0x0}, ...)
	github.com/ledgerwatch/erigon/polygon/bor/bor.go:1026 +0x312
github.com/ledgerwatch/erigon/cmd/state/exec3.(*Worker).RunTxTaskNoLock(0xc000b43b08, 0xdc009456708)
	github.com/ledgerwatch/erigon/cmd/state/exec3/state.go:226 +0x864
github.com/ledgerwatch/erigon/eth/stagedsync.ExecV3({_, _}, _, {_, _}, _, {{0x1087b1370, 0xdc00810dc80}, 0x100000, {0x1, ...}, ...}, ...)
	github.com/ledgerwatch/erigon/eth/stagedsync/exec3.go:786 +0x2c69
github.com/ledgerwatch/erigon/eth/stagedsync.ExecBlockV3(_, {_, _}, {{_, _}, {_, _}, _}, _, {0x1087a6a70, ...}, ...)
	github.com/ledgerwatch/erigon/eth/stagedsync/stage_execute.go:288 +0x1ea
github.com/ledgerwatch/erigon/eth/stagedsync.SpawnExecuteBlocksStage(_, {_, _}, {{_, _}, {_, _}, _}, _, {0x1087a6a70, ...}, ...)
	github.com/ledgerwatch/erigon/eth/stagedsync/stage_execute.go:395 +0x106
github.com/ledgerwatch/erigon/cmd/integration/commands.stageExec({0x1087b1370, 0xdc00810dc80}, {0x1087a6a70, 0xc0000eba40}, {0x1087b0c68, 0xc0004fab80})
	github.com/ledgerwatch/erigon/cmd/integration/commands/stages.go:1079 +0xcbe
github.com/ledgerwatch/erigon/cmd/integration/commands.init.func23(0x109398180, {0x107f4f16c?, 0x4?, 0x107f4eff4?})
	github.com/ledgerwatch/erigon/cmd/integration/commands/stages.go:203 +0x2d5
github.com/spf13/cobra.(*Command).execute(0x109398180, {0xc00097a7e0, 0x7, 0x7})
	github.com/spf13/cobra@v1.8.0/command.go:987 +0xab1
github.com/spf13/cobra.(*Command).ExecuteC(0x109395660)
	github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/spf13/cobra.(*Command).ExecuteContext(0xc0009efee8?, {0x1087a6a70?, 0xc0000eba40?})
	github.com/spf13/cobra@v1.8.0/command.go:1032 +0x47
main.main()
	github.com/ledgerwatch/erigon/cmd/integration/main.go:34 +0xe6
```

probably related with: https://github.com/ledgerwatch/erigon/issues/10969

--- a/eth/backend.go
+++ b/eth/backend.go
@@ -553,7 +553,7 @@ func New(ctx context.Context, stack *node.Node, config *ethconfig.Config, logger
 	}
 
 	var heimdallClient heimdall.HeimdallClient
-	var polygonBridge *bridge.Bridge
+	var polygonBridge bridge.Service
 
 	if chainConfig.Bor != nil {
 		if !config.WithoutHeimdall {

--- a/polygon/bor/bor.go
+++ b/polygon/bor/bor.go
@@ -329,7 +329,7 @@ type Bor struct {
 	frozenSnapshotsInit sync.Once
 	rootHashCache       *lru.ARCCache[string, string]
 	headerProgress      HeaderProgress
-	polygonBridge       bridge.Service
+	polygonBridge       bridge.PolygonBridge
 }
 
 type signer struct {

--- a/polygon/bor/bor.go
+++ b/polygon/bor/bor.go
@@ -329,7 +329,7 @@ type Bor struct {
 	frozenSnapshotsInit sync.Once
 	rootHashCache       *lru.ARCCache[string, string]
 	headerProgress      HeaderProgress
-	polygonBridge       bridge.PolygonBridge
+	polygonBridge       bridge.Service
 }
 
 type signer struct {
@@ -1485,7 +1485,7 @@ func (c *Bor) CommitStates(
 ) error {
 	blockNum := header.Number.Uint64()
 
-	if c.polygonBridge.(*bridge.Bridge) != nil {
+	if c.polygonBridge != nil {
 		events, err := c.polygonBridge.GetEvents(c.execCtx, blockNum)
 		if err != nil {
 			return err

